File: 5 - Linguagens de Programação/Atividades/02 - Incremento/main.dart
Function incremento(int valorInicial) {
  int atual = 0;
  return () {
    atual += valorInicial;
    return atual;
  };
}

void main() {
  var proximo = incremento(2);
  print(proximo());
  print(proximo());
  print(proximo());
  print(proximo());
}

File: 5 - Linguagens de Programação/Atividades/02 - Incremento/main.js
function incremento(valorInicial) {
    let atual = 0;
    return function() {
      atual += valorInicial;
      return atual;
    };
  }
  
  const proximo = incremento(2);
  console.log(proximo());
  console.log(proximo());
  console.log(proximo());
  console.log(proximo());
  